playground/TestShop/TestShop.AppHost/Program.cs

```diff
@@ -1,9 +1,10 @@
 var builder = DistributedApplication.CreateBuilder(args);
 
-var catalogDb = builder.AddPostgres("postgres")
-                       .WithDataVolume()
-                       .WithPgAdmin()
-                       .AddDatabase("catalogdb");
+var postgres = builder.AddPostgres("postgres")
```

Comment:
Why this change?

Reply:
cleaned up. this was left over from my initial explorations when I was using TestShop as a playground.